ngraph/frontend/onnx_editor/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "onnx_editor")
+
+file(GLOB_RECURSE LIBRARY_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp)
+file(GLOB_RECURSE LIBRARY_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/src/*.hpp)
+file(GLOB_RECURSE LIBRARY_PUBLIC_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/include/*.hpp)
```

Comment:
I prefer to explicitly list required files.

Reply:
The same as response to @tsocha.
I am also rather fan of using explicitly list of required files (especially if we generate solution for Window), but it is general convention in OV (for example `onnx_importer` had such explicitly list some time ago and it was changed).